index.html

```diff
@@ -40,7 +40,7 @@ <h1> Allen's CSD120 Website! </h1>
         <p>Every day the level our technology is evoking everyone the world. I have been impressed me a lot in different ways. So I choose this program to get more idea and knowledg about the programming.</p>
     </section> 
     
-    <section>
+    <section class="favourites">
```

Comment:
You need a way to identify this section so you can make it clear the floated mugshot